jakarta/quarkus3.yml

```diff
@@ -227,6 +229,64 @@ recipeList:
       oldFullyQualifiedTypeName: javax.annotation.sql.DataSourceDefinitions
       newFullyQualifiedTypeName: jakarta.annotation.sql.DataSourceDefinitions
 
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.jakarta.JavaxAuthenticationMigrationToJakartaAuthentication
+displayName: Migrate deprecated `javax.security.auth.message` packages to `jakarta.security.auth.message`
+description: Java EE has been rebranded to Jakarta EE, necessitating a package relocation.
+tags:
+  - authentication
+  - security
+  - javax
+  - jakarta
+
+recipeList:
+  - org.openrewrite.maven.AddDependency:
+      groupId: jakarta.authentication
+      artifactId: jakarta.authentication-api
+      version: 2.x
+      onlyIfUsing: javax.security.auth.message..*
```

Comment:
I don't think this is a good idea. You should use https://docs.openrewrite.org/reference/recipes/maven/changedependencygroupidandartifactid instead. And you shouldn't enforce the version as it will come from the BOM.

You might have to remove the version if set though. Not sure if it's doable with OpenRewrite atm.